src/pages/settings/Report/ReportSettingsPage.js

```diff
@@ -84,6 +84,8 @@ class ReportSettingsPage extends Component {
         const linkedWorkspace = _.find(this.props.policies, (policy) => policy && policy.id === this.props.report.policyID);
         const shouldDisableRename = this.shouldDisableRename(linkedWorkspace) || ReportUtils.isThread(this.props.report);
         const notificationPreference = this.props.translate(`notificationPreferencesPage.notificationPreferences.${this.props.report.notificationPreference}`);
+        const writeCapability = this.props.translate(`writeCapabilityPage.writeCapability.${this.props.report.writeCapability}`);
```

Comment:
I thinkkkk you might need to default `this.props.report.writeCapability` to `all` here too, right?

So maybe you could do something like:

```suggestion
        const writeCapability = this.props.report.writeCapability || CONST.WRITE_CAPABILITIES.ALL;
        const writeCapabilityText = this.props.translate(`writeCapabilityPage.writeCapability.${writeCapability}`);
```

so that down below you can use `writeCapability` without having to check if it's empty?